src/charm.py

```diff
@@ -117,6 +117,12 @@ def _oidc_layer(self):
                     "command": "/home/authservice/oidc-authservice",
                     "environment": self.service_environment,
                     "startup": "enabled",
+                    # working-dir is required to interchangeably support docker images and rocks.
+                    # Rocks always set their entrypoint working-dir to "/" because pebble is the
+                    # entrypoint, so we need to be explicit.  This was not needed for running
+                    # the upstream docker image because that image has a entrypoint working-dir
+                    # of "/home/authservice", which is used if working-dir is not set here.
```

Comment:
Could you update the comment because at the moment it kind of suggests that we added `working-dir` in order for charm to work with a ROCK?

Reply:
Resolved by [this](https://github.com/canonical/oidc-gatekeeper-operator/pull/128#discussion_r1459293412)